Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.h

```diff
@@ -39,8 +39,28 @@ namespace AZ::Render
     class AtomActorDebugDraw
     {
     public:
+        struct TrajectoryPathParticle
+        {
+            EMotionFX::Transform m_worldTm;
+        };
+
+        struct TrajectoryTracePath
+        {
+            AZStd::vector<TrajectoryPathParticle> m_traceParticles;
+            const EMotionFX::ActorInstance* m_actorInstance;
+            float m_timePassed;
+
+            TrajectoryTracePath()
+            {
+                m_traceParticles.reserve(250);
+                m_timePassed = 0.0f;
+                m_actorInstance = NULL;
```

Comment:
update: Likely best to move these to member initializers (and use `nullptr` instead of `NULL`). Also might be worth removing `m_traceParticles` reserve and let the calling code handle it instead of forcing the user to have to have this value (or let them pass in a number to the constructor?)

Reply:
hmm, yeah I'll just get rid of it and let user decide what to reserve.